### PR TITLE
Add apertis

### PR DESCRIPTION
**Pull request by jjardon**
Census: https://wiki.debian.org/Derivatives/Census/Apertis
Homepage: https://www.apertis.org/

### CODE REVIEW

File: repos.d/deb/apertis.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+###########################################################################
+# Apertis
+###########################################################################
+- name: apertis_2020
+  type: repository
+  desc: Trisquel 2020
```
Comment (AMDmi3) [inline repos.d/deb/apertis.yaml:6]: Please use correct distro name here, `v` prefix as well if it's reasonable.

File: repos.d/deb/apertis.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+###########################################################################
+# Apertis
+###########################################################################
+- name: apertis_2020
```
Comment (AMDmi3) [inline repos.d/deb/apertis.yaml:4]: Shouldn't it be `v2020`? Looks like version is mentioned with `v` on the site.

File: repos.d/deb/apertis.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  type: repository
+  desc: Trisquel 2020
+  statsgroup: Debian+derivs
+  family: debuntu
+  color: '669fdd'
+  minpackages: 20000
+  sources:
+    - name: [ v2020/development, v2020/target, v2020/sdk v2020/hmi, v2020-updates/development, v2020-updates/target, v2020-updates/sdk, v2020-updates/hmi ]
```
Comment (AMDmi3) [inline repos.d/deb/apertis.yaml:12]: Comma missing between sdk and hmi

File: repos.d/deb/apertis.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+- name: apertis_2021
+  type: repository
+  desc: Trisquel 2021 Development
+  statsgroup: Debian+derivs
+  family: debuntu
+  color: '669fdd'
+  minpackages: 20000
+  sources:
+    - name: [ v2021dev3/development, v2021dev3/target, v2021dev3/sdk v2021dev3/hmi ]
```
Comment (AMDmi3) [inline repos.d/deb/apertis.yaml:31]: Comma missing here as well

File: repos.d/deb/apertis.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
+###########################################################################
+# Apertis
+###########################################################################
+- name: apertis_2020
+  type: repository
+  desc: Trisquel 2020
+  statsgroup: Debian+derivs
+  family: debuntu
+  color: '669fdd'
```
Comment (AMDmi3) [inline repos.d/deb/apertis.yaml:9]: May use `3a5a80` here is it's the color used for the logo on the site.

File: repos.d/deb/apertis.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  minpackages: 20000
```
Comment (AMDmi3) [inline repos.d/deb/apertis.yaml:10]: The repo doesn't have 20k packages, so this should be around 3500.

File: repos.d/deb/apertis.yaml [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+      parser: DebianSourcesParser
+      url: 'https://repositories.apertis.org/apertis/dists/{source}/source/Sources.gz'
+      compression: gz
+      subrepo: '{source}'
+  repolinks:
+    - desc: Apertis home
+      url: https://www.apertis.org
+  tags: [ all, production, apertis ]
+
+- name: apertis_2021
```
Comment (AMDmi3) [inline repos.d/deb/apertis.yaml:23]: Same changes as with 2020 repo are needed here.

**Review by AMDmi3 — CHANGES_REQUESTED** [review]

**Comment by jjardon** [comment]
@AMDmi3 Thanks for the quick review and sorry for all the little mistakes; I think everything is ok now

**Comment by AMDmi3** [comment]
Deployed, thanks!

**Pull request merged**